weblate/static/style-bootstrap.css

```diff
@@ -417,6 +417,12 @@ kbd {
 .unit-navi {
     min-width: 25em;
 }
+.unit-navi li {
+    float: left;
+}
```

Comment:
JFYI: This broke rendering in RTL languages, see https://github.com/WeblateOrg/weblate/issues/3788, fixed by a3adbe9f58f173d222f7d0458758565696eb784f

Reply:
Thanks for letting me know. Will keep in mind next time.

Reply:
Basically anything asymmetric in the CSS (padding, margins, float, align,..) needs to be handled for RTL languages as well.

Reply:
Got it. I should have thought of that.